geometric/run_json.py

```diff
@@ -62,16 +62,28 @@ def get_output_json_dict(in_json_dict, schema_traj):
     })
     return out_json_dict
 
+def make_constraints_string(constraints_dict):
+    """ Convert the new constraints dict format into the original string format """
+    constraints_string = ''
+    for key, value_list in constraints_dict.items():
+        if key not in ('freeze', 'set', 'scan'):
+            print("Warning, constraints key %s is not recognized" % key)
```

Comment:
This should probably raise a KeyError to avoid some rather fundamental errors.

Reply:
Done